src/main/java/frc/robot/commands/AutoBalance.java

```diff
@@ -47,39 +47,48 @@ public void initialize() {
   @Override
   public void execute() {
     // Gets how far the robot is tilted and calculates proper drive power from it
-    m_currentAngle = m_drive.getGyroRoll().getDegrees();
+    m_currentAngle = m_drive.getGyroPitch().getDegrees();
     m_error = AutoConstants.DESIRED_BALANCE_ANGLE - m_currentAngle;
 
     m_drivePower = Math.min(m_balanceController.calculate(m_currentAngle), 1);
     // Limit max power
-    m_drivePower = MathUtil.clamp(m_drivePower, -0.2, 0.2);
+    m_drivePower = MathUtil.clamp(m_drivePower, -0.6, 0.6);
+
+    double rollRateBalance = m_drive.getGyroRollPitch();
+    if (Math.abs(rollRateBalance) >= 0.3) {
+      m_drivePower = Math.copySign(0.0, -m_drive.getGyroPitch().getDegrees());
```

Comment:
Math.copySign doesn't do anything useful here. Why are we looking at roll? More documentation would be nice for this command.

Reply:
We are looking at roll as that is the value that changes when the robot is attempting to auto balance. I will add more documentation now before pulling.

Reply:
The reason that it is at 0.0 was because I was experimenting with  moving in a negative direction when it sees that it's tilting forward, to stop from over-shooting the charge station